Library/Homebrew/dependency_collector.rb

```diff
@@ -121,6 +121,7 @@ def parse_symbol_spec(spec, tags)
     when :x11        then X11Requirement.new(spec.to_s, tags)
     when :xcode      then XcodeRequirement.new(tags)
     when :linux      then LinuxRequirement.new(tags)
+    when :skip_linux then SkipLinuxRequirement.new(tags)
```

Comment:
I'd suggest moving this line up one or down one so that `:linux` and `:macos` are together.

Reply:
done